src/app/navbar/navbar.component.html

```diff
@@ -3,24 +3,36 @@
         <mat-icon>book</mat-icon>
     </button>
 
-    <span class="font-weight-light mr-3">Glasgow University Book Society</span>
+    <a routerLink=""><span class="d-none d-sm-inline font-weight-light mr-3">Glasgow University Book Society</span></a>
+    <a routerLink=""><span class="d-sm-none font-weight-light mr-3">GU Book Society</span></a>
 
-
-    <a routerLink=""><span class="d-none d-lg-inline font-weight-light nav-link pb-1">Link 1</span></a>
     <a routerLink="/form"><span class="d-none d-lg-inline font-weight-light nav-link pb-1">Form Page</span></a>
-    <a routerLink="404"><span class="d-none d-lg-inline font-weight-light nav-link pb-1">Link 3</span></a>
+    <a routerLink="404"><span class="d-none d-lg-inline font-weight-light nav-link pb-1">404 Page</span></a>
 
     <span class="fill-remaining-space"></span>
 
     <div class="d-none d-lg-inline">
-        <button mat-button class="mr-1" id="signIn"><span>Sign In</span></button>
-        <button mat-button id="signUp"><span>Sign Up</span></button>
+        <button mat-button class="mr-1" id="signIn" routerLink="signIn"><span>Sign In</span></button>
+        <button mat-button id="signUp" routerLink="signUp"><span>Sign Up</span></button>
     </div>
 
     <div class="d-lg-none">
-        <button mat-icon-button color="secondary" aria-label="Example icon button with a menu icon">
+        <button mat-icon-button color="secondary" data-toggle="collapse" data-target="#navbarToggleExternalContent" aria-label="Toggle navigation">
             <mat-icon>menu</mat-icon>
         </button>
     </div>
 
 </mat-toolbar>
+
+<div class="d-lg-none collapse text-center" id="navbarToggleExternalContent">
+    <div id="dropdown" class="px-4 pb-4 pt-2">
+        <a routerLink="/form"><span class="h6">Form Page</span></a>
+        <hr class="nav-divider">
+        <a routerLink="/404"><span class="h6">404 Page</span></a>
+        <hr class="nav-divider">
+        <div class="text-center">
+            <button mat-button class="ml-0 mr-2 collapsedButton" routerLink="signIn"><span>Sign In</span></button>
+            <button mat-button class="collapsedButton" routerLink="signUp"><span>Sign Up</span></button>
+        </div>
+    </div>
+</div>
```

Comment:
Tidied up the navbar and added the code to create a dropdown collapsible/expandable nav section on mobiles.